src/demo/renderer.js

```diff
@@ -498,9 +504,75 @@ const drawPredictBot = state => {
   ctx.drawImage(botImg, botX, botY);
 };
 
+const drawWordFishImages = () => {
+  const canvas = getState().canvas;
+  const ctx = canvas.getContext('2d');
+  const state = getState();
+
+  const fishScale = 0.7;
+
+  const t = $time();
+  state.wordFish.forEach(fish => {
+    const swayValue = ((t * 360) / (20 * 1000)) % 360;
+    const swayOffsetY = Math.sin(((swayValue * Math.PI) / 180) * 3) / 20;
```

Comment:
At some point we might want to factor out a wobble helper function :)

Reply:
+1 I did just copy and adjust these from the pond.